modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/download-report/DownloadPDFReport.tsx

```diff
@@ -226,6 +232,7 @@ const getContainers = async (
 const DownloadPDFReport: React.FC<IDownloadReport> = ({
 	dateRangeDescription,
 	disabled,
+	eventAnalysisDownloadReport = false,
 	showDateRange,
 	subtitle,
 	title,
```

Comment:
Create a new property `infoMessage`:

infoMessage = Liferay.Language.get(
						'the-dashboard-will-be-downloaded-exactly-as-it-is-displayed-on-your-screen.-please-verify-if-the-desired-tabs-and-filters-are-selected-before-proceeding'
					)

Reply:
done